backend/app/extraction/TranscriptionExtractor.scala

```diff
@@ -43,7 +59,8 @@ class TranscriptionExtractor(index: Index, scratchSpace: ScratchSpace, ingestion
     println(params.languages)
 
     val result = Either.catchNonFatal{
-      val transcriptResult = Whisper.invokeWhisper(file.toPath, tmpDir, translate =false)
+      val convertedFile = FfMpeg.convertToWav(file.toPath, ffMpegTmpDir)
```

Comment:
It sounds like conversion is faster than transcription but would it be worth skipping this if the file's already a wav?